metricbeat/module/elasticsearch/shard/data.go

```diff
@@ -72,7 +72,7 @@ func eventsMapping(r mb.ReporterV2, content []byte, isXpack bool) error {
 					ModuleFields: common.MapStr{},
 				}
 
-				event.ModuleFields.Put("cluster.state.id", stateData.StateID)
+				event.ModuleFields.Put("cluster.stats.state.state_uuid", stateData.StateID)
```

Comment:
This is breaking change at the moment. Instead of removing the old field, add a new one duplicating the value of the current one.

At the same time, we need to add this new field in the corresponding `fields.yml`, this one: https://github.com/elastic/beats/blob/master/metricbeat/module/elasticsearch/cluster_stats/_meta/fields.yml

The run a `mage fmt update` at the root of metricbeat folder and you'll see a bunch of files changed.

Reply:
Isn't this field [already defined](https://github.com/elastic/beats/blob/master/metricbeat/module/elasticsearch/cluster_stats/_meta/fields.yml#L18) or am I looking at wrong place ?

Reply:
It is already defined, yes. But that just mean "index it" maybe it was a leftover and we finally don't write into that field in the module.